src/components/site/SiteNavigation.md

```diff
@@ -0,0 +1,25 @@
+```jsx
```

Comment:
This is just showing up with all of it's styles in the design system, so I think we should remove for now.

![image](https://user-images.githubusercontent.com/1933400/68154648-f74e2300-ff15-11e9-8f1b-1b136be10cd9.png)

Reply:
especially since it's a single use component.

Reply:
removed.